common/python/cctrusted_base/quote.py

```diff
@@ -2,41 +2,39 @@
 Quote data structures
 """
 
+import logging
 from abc import ABC, abstractmethod
+from cctrusted_base.binaryblob import BinaryBlob
 
 
 
-class QuoteHeader(ABC):
+LOG = logging.getLogger(__name__)
+
+
+
+class QuoteHeader(BinaryBlob):
     """
     Quote Header abstract class (interface)
     """
 
-    @abstractmethod
-    def get_data(self) -> bytearray:
-        """
-        Get raw data
-        """
-        raise NotImplementedError("Should be implemented by inherited class")
-
-class QuoteBody(ABC):
+class QuoteBody(BinaryBlob):
     """
     Quote Body abstract class (interface)
     """
 
-    @abstractmethod
-    def get_data(self) -> bytearray:
-        """
-        Get raw data
-        """
-        raise NotImplementedError("Should be implemented by inherited class")
-
-QuoteSignature = bytearray
+class QuoteSignature(BinaryBlob):
+    """
+    Quote Signature
+    """
 
 class Quote(ABC):
     """
     Quote abstract class (interface)
     """
 
+    QUOTE_OUT_FORMAT_RAW = "raw"
+    QUOTE_OUT_FORMAT_HUMAN = "human"
+
     @abstractmethod
     def get_header(self) -> QuoteHeader:
```

Comment:
Is this applicable for TPM quote?

Reply:
Sorry for the confusing. Please let me clarify:
They are not defined by TPM. These format are used for dump() that is defined in our code.
I think the definition can be shared by TPM Quote dump and TDX Qute dump.

Reply:
As discussion, I have removed these literal definitions from this top-level base class to avoid confusion.
Please review the updated version. Thanks!